JAICore/jaicore-search/src/test/java/jaicore/search/algorithms/standard/bestfirst/RandomCompletionNodeEvaluatorTester.java

```diff
@@ -64,22 +65,22 @@ public void testDuplicateFreeEnumeration() throws NodeEvaluationException, Inter
 		/* this is cheap, test it several times */
 		for (int seed = 0; seed < 5; seed++) {
 			List<List<QueenNode>> seenSolutions = new ArrayList<>();
-			RandomCompletionBasedNodeEvaluator<QueenNode, Double> ne = getSeededNodeEvaluator(seed, 20); // draw (up to
-																											// 20
-																											// examples),
-																											// but only
-																											// 10
-																											// solutions
-																											// exist
+			RandomCompletionBasedNodeEvaluator<QueenNode, Double> ne = this.getSeededNodeEvaluator(seed, 20); // draw (up to
```

Comment:
maybe we can reduce this comment to a single line or two

Reply:
yep